src/js/parsers/post.js

```diff
@@ -52,7 +52,7 @@ export default {
 
     // walk up from the textNode until the rootNode, converting each
     // parentNode into a markup
-    function collectMarkups(textNode, rootNode) {
+    const  collectMarkups = (textNode, rootNode) =>{
```

Comment:
I think when creating functions to use in blocks (i.e. helper functions in other functions), it makes some sense to declare them in a variable-ish way like this. Mostly, though, I think I just enjoy the new syntax and I convert named function expressions over to this if I stare at them too long :P